search_service/api/table.py

```diff
@@ -18,6 +23,8 @@
     "column_names": fields.List(fields.String),
     # tags can be empty list
     "tags": fields.List(fields.String),
```

Comment:
in that case, should we handle tags the same way as badge as well?

Reply:
yeah, i figured that would be best!